spec/migrations/20150903162623_assign_tenant_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+require_migration
+
+describe AssignTenant do
+  let(:tenant_stub)           { migration_stub(:Tenant) }
+  let(:ems_stub)              { migration_stub(:ExtManagementSystem) }
+  let(:miq_ae_namespace_stub) { migration_stub(:MiqAeNamespace) }
+  let(:miq_group_stub)        { migration_stub(:MiqGroup) }
+  let(:provider_stub)         { migration_stub(:Provider) }
+  let(:tenant_quota_stub)     { migration_stub(:TenantQuota) }
+  let(:vm_stub)               { migration_stub(:Vm) }
+
+  let(:stubs) { [ems_stub, miq_ae_namespace_stub, miq_group_stub, provider_stub, tenant_quota_stub, vm_stub] }
+
+  migration_context :up do
+    describe "#tenant0" do
```

Comment:
Should this say root_tenant or something?  `tenant0` is not really a term in our product, just something we keep throwing around.